cmd/containerd-shim-runhcs-v1/service.go

```diff
@@ -98,405 +97,150 @@ func NewService(o ...ServiceOption) (svc *service, err error) {
 }
 
 func (s *service) State(ctx context.Context, req *task.StateRequest) (resp *task.StateResponse, err error) {
-	ctx, span := oc.StartSpan(ctx, "State")
-	defer span.End()
-	defer func() {
-		if resp != nil {
-			span.AddAttributes(
-				trace.StringAttribute("status", resp.Status.String()),
-				trace.Int64Attribute("exitStatus", int64(resp.ExitStatus)),
-				trace.StringAttribute("exitedAt", resp.ExitedAt.String()))
-		}
-		oc.SetSpanStatus(span, err)
-	}()
-
-	span.AddAttributes(
-		trace.StringAttribute("tid", req.ID),
-		trace.StringAttribute("eid", req.ExecID))
-
-	if s.isSandbox {
-		span.AddAttributes(trace.StringAttribute("pod-id", s.tid))
-	}
+	s.logEntry(ctx).Trace("State")
```

Comment:
so we're no longer logging request information?

Reply:
We do, we now add the entire request to the auto-generated span, instead of manually adding bits of the request to the individual spans we create here
https://github.com/microsoft/hcsshim/blob/593c1e79f184dcab224d2afb8cc4d4ad6dfbb765/pkg/octtrpc/interceptor.go#L122-L124